addon/components/frost-application-bar.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-application-bar'
```

Comment:
It is just a new wrapper that we use, nothing to do with the actual ember-frost-app-bar repo.